src/modules/SdnDiag.LoadBalancerMux/private/Get-SdnDipProbeInfoFromHost.ps1

```diff
@@ -0,0 +1,14 @@
+function Get-SdnDipProbeInfoFromHost {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [IPAddress[]]$HostIPAddress,
+
+        [Parameter(Mandatory = $false)]
+        [System.String]$ProbeID = $null
+    )
+
+    $slbManager = Connect-SlbManager
```

Comment:
This does not pass SlbManagerPrimary is it by purpose to make it run on SlbManagerPrimary or we missed? There is no reference yet though.